packages/postgres/src/PostgresMessagePickupRepository.ts

```diff
@@ -0,0 +1,569 @@
+import {
+  AddMessageOptions,
+  Agent,
+  GetAvailableMessageCountOptions,
+  injectable,
+  Logger,
+  MessagePickupEventTypes,
+  MessagePickupLiveSessionRemovedEvent,
+  MessagePickupLiveSessionSavedEvent,
+  MessagePickupRepository,
+  QueuedMessage,
+  RemoveMessagesOptions,
+  TakeFromQueueOptions,
+} from '@credo-ts/core'
+import { Pool, Client } from 'pg'
+import PGPubsub from 'pg-pubsub'
+import * as os from 'os'
+import { createTableMessage, createTableLive, messagesTableName, liveSessionTableName } from '../config/dbCollections'
+import { ConnectionInfo, PostgresMessagePickupRepositoryConfig } from './interfaces'
+import { MessagePickupSession } from '@credo-ts/core/build/modules/message-pickup/MessagePickupSession'
+import axios from 'axios'
+
+@injectable()
+export class PostgresMessagePickupRepository implements MessagePickupRepository {
+  private logger?: Logger
+  public messagesCollection?: Pool
```

Comment:
Any specific reason for this to be public? Otherwise:

```suggestion
  private messagesCollection?: Pool
```

Reply:
Solved